.gitignore

```diff
@@ -1,3 +1,5 @@
+*.cache
```

Comment:
Why is this needed?

Reply:
I noticed when running locally that a cache file is created (`twisted/plugins/dropin.cache`), didn't want to accidentally commit it